messages/javascript/src/NdjsonToMessageStream.ts

```diff
@@ -43,7 +43,7 @@ export default class NdjsonToMessageStream<T> extends Transform {
         const object = JSON.parse(this.buffer)
         this.push(this.fromObject(object))
       } catch (err) {
-        return callback(err)
+        return callback(new Error(`Not JSONs: ${this.buffer}`))
```

Comment:
Perhaps also include the original error? That way we know what the parser was complaining about.

Reply:
That's not easily done in JavaScript - exceptions can't have references to other exceptsions like in Java. I don't think it's needed either.